### ISSUE TITLE
does epoch_size para NOT worked in model.FeedForward interface??

### ISSUE THREAD

**Issue by beyondli** [apache/incubator-mxnet#6970]
when train my data with model.FeedForward, I set epoch_size =100, and set batch_size=16. so I think the ONE EPOCHS should just contained 1600 samples though in real train dataset the sample number is very bigger than 1600.(the main reason that my want control the save frequency)
When I set epoch_end_callback to save model, I hope when 1600 samples after, the model could be saved for one time,but when I tested ,I found no save is called ,just like followed ... lr could be updated with my set ,but nothing saved ...,so Does the epoch_size is not worked at all in this interface that has been depreciated??
any help is very appreciated!!

2017-07-09 22:21:25,866 Node[0.] Epoch[0] Batch [50]	Speed: 79.13 samples/sec	
2017-07-09 22:21:36,195 Node[0.] Epoch[0] Batch [100]	Speed: 77.46 samples/sec	
2017-07-09 22:21:36,195 Node[0.] Update[101]: Change learning rate to 1.00000e-03
2017-07-09 22:21:46,506 Node[0.] Epoch[0] Batch [150]	Speed: 77.58 samples/sec	
2017-07-09 22:21:56,875 Node[0.] Epoch[0] Batch [200]	Speed: 77.16 samples/sec	a
2017-07-09 22:21:56,876 Node[0.] Update[201]: Change learning rate to 1.00000e-04

**Comment by szha** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to ping me to reopen if this is still an active issue. Thanks!
